voice/voice-ai/x/OpenAIService/OpenAIUtils.swift

```diff
@@ -0,0 +1,50 @@
+
+import Foundation
+
+struct OpenAIUtils {
+    static func limitConversationContext (_ conversaton: [Message], charactersCount: Int) -> [Message] {
```

Comment:
Usually the bot says a lot more than the user. Perhaps we should only take a small part of each response by the bot so it preserves a longer context

Reply:
i agree with you, also we need a balance between quality and cost.

Perhaps it is worth taking all user requests without any changes and only shorten the bot's responses?

Reply:
Yes, I think that would capture a lot more, until we come up with better solution